### PR TITLE
CSP-1184 Upgrade .NET6 to .NET8

### PR DESCRIPTION
**Pull request by ManishAgarwalDfE**

### CODE REVIEW

File: src/SFA.DAS.Roatp.ProviderModeration.Web/Controllers/ErrorController.cs [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+public class ErrorController : Controller
+{
+    private readonly ILogger<ErrorController> _logger;
+
+    public ErrorController(ILogger<ErrorController> logger)
+    {
+        _logger = logger;
+    }
+
+    [AllowAnonymous]
```
Comment (MarkFCain) [inline src/SFA.DAS.Roatp.ProviderModeration.Web/Controllers/ErrorController.cs:18]: you can just put [AllowAnonymous] at the top of the controller and remove them from the mehods.  Smoke test it though.   I'll leave it up to you

File: src/SFA.DAS.Roatp.ProviderModeration.Web/Controllers/ErrorController.cs [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+    {
+        var feature = HttpContext.Features.Get<IExceptionHandlerPathFeature>();
+        ErrorViewModel errorViewModel = new()
+        {
+            HomePageUrl = Url.RouteUrl(RouteNames.GetProviderDescription)!
+        };
+
+        if (User.Identity!.IsAuthenticated)
+        {
+            _logger.LogError(feature!.Error, "Unexpected error occured during request to path: {path} by user: {user}", feature.Path, HttpContext.User.FindFirstValue(ClaimTypes.Upn));
```
Comment (MarkFCain) [inline src/SFA.DAS.Roatp.ProviderModeration.Web/Controllers/ErrorController.cs:48]: is it usual for us to log the user so explicitly?  I would just log the upn or something similarly anonymous

Comment (ManishAgarwalDfE) [reply src/SFA.DAS.Roatp.ProviderModeration.Web/Controllers/ErrorController.cs:48]: Yes, I made the same changes in the Apprentice and Employer web

File: src/SFA.DAS.Roatp.ProviderModeration.Web/Controllers/ErrorController.cs (hunk 1 above)
Comment (ManishAgarwalDfE) [reply src/SFA.DAS.Roatp.ProviderModeration.Web/Controllers/ErrorController.cs]: Done

**Review by MarkFCain — APPROVED** [review]

**Pull request merged**